src/main/java/teammates/common/datatransfer/FeedbackConstantSumQuestionDetails.java

```diff
@@ -524,11 +524,18 @@ public String getQuestionResultStatisticsCsv(
         
         for(Entry<String, Integer[]> entry : optionTotalCount.entrySet() ){
             double average = entry.getValue()[0]/entry.getValue()[1];
-            fragments += entry.getKey() + ","
-                      + df.format(average) + Const.EOL;
+            if(distributeToRecipients){
+                String teamName = bundle.getTeamNameForEmail(entry.getKey());
+                String recipientName = bundle.getNameForEmail(entry.getKey());
+                fragments +=teamName+","+recipientName+","+df.format(average)+Const.EOL;
+            }
+            else{
```

Comment:
Sorry for not pointing this out earlier, but can you shift the 'else' up so that it is on the same line as the curly brackets? 
Have a look at our coding standards for Java: https://docs.google.com/document/pub?id=1iAESIXM0zSxEa5OY7dFURam_SgLiSMhPQtU0drQagrs&embedded=true#h.156j9wx7tamx